presto-main/src/main/java/com/facebook/presto/sql/planner/PlanPrinter.java

```diff
@@ -323,6 +331,11 @@ private static String formatFragment(Metadata metadata, Session session, PlanFra
         return builder.toString();
     }
 
+    private static boolean isNonZero(DataSize dataSize)
+    {
+        return dataSize != null && dataSize.getValue() != 0;
```

Comment:
it can't, it was a reflex to implement a static helper method as null safe, I can change that if you want